### PR TITLE
docs(kitex): translate code-gen and fix typo

### PR DESCRIPTION
**Pull request by Nihilism0**
What type of PR is this?
docs

What this PR does / why we need it (en: English/zh: Chinese):
en: translate kitex code-gen doc based on Chinese doc and fix typo
zh: 中文翻译kitex code-gen文档和改正中文文档错字

### CODE REVIEW

**Comment by Nihilism0** [comment]
this issue#[861](https://github.com/cloudwego/kitex/issues/861)

**Review by HeyJavaBean — DISMISSED** [review]

File: content/en/docs/kitex/Tutorials/code-gen/code_generation.md [hunk 1]
```diff
@@ -1,63 +1,138 @@
...
+Kitex code generation relies on thriftgo and protoc compilers. You need to install the corresponding compilers first: [thriftgo](https://github.com/cloudwego/thriftgo) or [protoc](https://github.com/protocolbuffers/protobuf/releases).
+
+After installing the aforementioned tools, you can install the Kitex command-line tool itself using the following go command:
+
+```shell
 go install github.com/cloudwego/kitex/tool/cmd/kitex@latest
 ```
 
-You can use the go command to install kitex or build and install it from source. To check where kitex will be installed, try
+Alternatively, you can download the Kitex source code and navigate to the tool/cmd/kitex directory. From there, you can execute go install to install the Kitex command-line tool:
```
Comment (GuangmingLuo) [inline content/en/docs/kitex/Tutorials/code-gen/code_generation.md:24]: 对齐下中文文档，`tool/cmd/kitex` 和 `go instal`l 都加一下代码块 block，下文同

Comment (Nihilism0) [reply content/en/docs/kitex/Tutorials/code-gen/code_generation.md]: sry,please recheck

File: 内容/zh/文档/风筝/教程/代码生成/code_generation.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ description: >
```
Comment (HeyJavaBean) [inline 内容/zh/文档/风筝/教程/代码生成/code_generation.md:13]: 文件目录为啥都变成中文了

File: content/en/docs/kitex/Tutorials/code-gen/code_generation.md [hunk 3]
```diff
@@ -1,63 +1,138 @@
...
+├── kitex_info.yaml  	// Record meta information for integration with cwgo tools
+├── main.go		 	 // Quickly start the main function of the server
+└── script			 // Build service related scripts
+│    └── bootstrap.sh
+├── kitex_gen
+     └── ....	 
+    
 ```
 
+After filling in the business code in the interface of `handler. go`, execute the main function of `main. go` to quickly start the Kitex Server.
```
Comment (HeyJavaBean) [inline content/en/docs/kitex/Tutorials/code-gen/code_generation.md:108]: in the interface of `handler. go` 改成 in the generated function of `handler. go`  可能会合适一些

File: 内容/zh/文档/风筝/教程/代码生成/code_generation.md (hunk 2 above)
Comment (Nihilism0) [reply 内容/zh/文档/风筝/教程/代码生成/code_generation.md]: sry,plz recheck

**Review by HeyJavaBean — APPROVED** [review]

**Pull request merged**